test/e2e/tests/request-queuing/ui.spec.js

```diff
@@ -430,14 +430,24 @@ describe('Request-queue UI changes', function () {
         // Open the second dapp and switch chains
         await openDappAndSwitchChain(driver, DAPP_ONE_URL, '0x1');
 
+        // await driver.delay(500000);
+
         // Go to wallet fullscreen, ensure that the global network changed to Ethereum Mainnet
         await driver.switchToWindowWithTitle(
           WINDOW_TITLES.ExtensionInFullScreenView,
         );
 
-        // Open Network Manager and delete custom network
         const networkManager = new NetworkManager(driver);
         await networkManager.openNetworkManager();
+        await networkManager.selectTab('Popular');
+        await networkManager.selectNetworkByChainId(NetworkId.LINEA);
+
+        await networkManager.openNetworkManager();
+        await networkManager.selectTab('Popular');
+        await networkManager.selectNetworkByChainId(NetworkId.ETHEREUM);
```

Comment:
What is the purpose of doing this in the test?

It looks like we open the network selector and select linea, then do the same for ethereum, then we finally select custom tab and delete the custom network.